app/src/androidTest/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPluginTest.kt

```diff
@@ -266,11 +281,11 @@ class LocationLayerPluginTest {
 
         uiController.loopMainThreadForAtLeast(MAP_RENDER_DELAY)
 
-        val point: Point = mapboxMap.queryLocationSourceFeatures(LOCATION_SOURCE)[0].geometry() as Point
+        val point: Point = mapboxMap.querySourceFeatures(LOCATION_SOURCE)[0].geometry() as Point
 
-        Assert.assertThat(plugin.locationEngine, nullValue())
-        Assert.assertEquals(point.latitude(), location.latitude, 0.1)
-        Assert.assertEquals(point.longitude(), location.longitude, 0.1)
+        assertThat(plugin.locationEngine, nullValue())
```

Comment:
is this equivalent to `assertNull(plugin.locationEgine)`?

Reply:
Yes, and since that was the approach taken with this test suite before I took over, I think I'm going to stick with it. Going to do some cleanups of those assertions though.